config/locales/en/mailers.yml

```diff
@@ -2,6 +2,9 @@ en:
   mailers:
     title: "Open Government"
     no_reply: "This message was sent from an email address that does not accept replies."
+    already_confirmed:
+      new_password: "You can reset your password at the following link:"
```

Comment:
I think this sentence isn't very clear in this context :thinking:. It might imply that resetting the password is necessary after confirming an email. Not to mention I might receive this message after someone else has asked to confirm my email :sweat_smile:.

How about something like (just an example):

> We've received a request to send you instructions to confirm your account. However, your account is already confirmed, so there's no need to do so again.
>
> If you've forgotten your password, you can reset it at (...)